drag_and_drop_v2/templates/html/js_templates.html

```diff
@@ -0,0 +1,121 @@
+            <label for="item-{{id}}-text">{{i18n "Text"}}</label>
+            <input type="text"
+                   id="item-{{id}}-text"
+                   class="item-text"
+                   value="{{ displayName }}" />
+            <label for="item-{{id}}-zone">{{i18n "Zone"}}</label>
+            <select id="item-{{id}}-zone"
+                    class="zone-select">{{ dropdown }}</select>
+            <a href="#" class="remove-item hidden">
+                <div class="icon remove"></div>
+            </a>
+        </div>
+        <div class="row">
+            <label for="item-{{id}}-image-url">{{i18n "Image URL (alternative to the text)"}}</label>
+            <input type="text"
+                   id="item-{{id}}-image-url"
+                   class="item-image-url"
+                   value="{{ imageURL }}" />
+        </div>
+        <div class="row">
+            <label for="item-{{id}}-image-description">{{i18n "Image description (should provide sufficient information to place the item even if the image did not load)"}}</label>
+            <textarea id="item-{{id}}-image-description"
+                      class="item-image-description">{{ imageDescription }}</textarea>
+        </div>
+        <div class="row">
+            <label for="item-{{id}}-success-feedback">{{i18n "Success Feedback"}}</label>
+            <textarea id="item-{{id}}-success-feedback"
+                      class="success-feedback">{{ feedback.correct }}</textarea>
+        </div>
+        <div class="row">
+            <label for="item-{{id}}-error-feedback">{{i18n "Error Feedback"}}</label>
+            <textarea id="item-{{id}}-error-feedback"
+                      class="error-feedback">{{ feedback.incorrect }}</textarea>
+        </div>
+        <div class="row advanced-link">
+            <a href="#">{{i18n "Show advanced settings" }}</a>
+        </div>
+        <div class="row advanced">
+            <label for="item-{{id}}-width-percent">{{i18n "Preferred width as a percentage of the background image width (or blank for automatic width):"}}</label>
+            <input type="number" id="item-{{id}}-width-percent" class="item-width" value="{{ singleDecimalFloat widthPercent }}" step="0.1" min="1" max="99" />%
+        </div>
+        <div class="row advanced">
+          <label for="item-{{id}}-numerical-value">
+            {{i18n "Optional numerical value (if you set this, students will be prompted for this value after dropping this item)"}}
+          </label>
+          <input type="number"
+                 step="0.1"
+                 id="item-{{id}}-numerical-value"
+                 class="item-numerical-value" value="{{ numericalValue }}" />
+        </div>
+        <div class="row advanced">
+          <label for="item-{{id}}-numerical-margin">
+            {{i18n "Margin ± (when a numerical value is required, values entered by students must not differ from the expected value by more than this margin; default is zero)"}}
```

Comment:
Change "students" to "learners."

"Margin ± (when a numerical value is required, values entered by learners must not differ from the expected value by more than this margin; default is zero)"